src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -34,6 +34,21 @@ enum {
                                                                uct_rdmacm_cm_ep_t::status.*/
 };
 
+
+static UCS_F_ALWAYS_INLINE
+uct_rdmacm_cm_t *uct_rdmacm_cm_ep_get_cm(uct_rdmacm_cm_ep_t *cep)
```

Comment:
maybe pass uct_cm_base_ep_t to UCT_CM_FAILURE_LOG macro?
anyway, if we have an error we'd like to hid, it has to be peer-related and not some generic error